test/xrp-client-integration-test.ts

```diff
@@ -70,7 +70,7 @@ describe('Xpring JS XRPClient Integration Tests', function(): void {
   > {
     this.timeout(timeoutMs)
 
-    const transactionStatus = await legacyXRPClientWeb.getTransactionStatus(
+    const transactionStatus = await legacyXRPClientWeb.getPaymentStatus(
```

Comment:
So even though we're doing breaking changes, we still have the `legacyXRPClient` around?

Should cutting that be part of `2.0`?

Reply:
This will get cut when we move to rippled 1.5. 

I'm waiting for two things to happen:
1) rippled 1.5 is actually released (it's currently just an RC)
2) we've had the backend of the wallet on rippled in prod for a week or two with no issues